sample/core.py

```diff
@@ -161,18 +171,18 @@ def get_categoryinfo(self):
                     continue
                 if page_id not in ci_list:
                     ci_list[page_id] = []
-                ci_list[page_id] = page_content[prop]
+                ci_list[page_id] += page_content[prop]
 
         self.payload.pop('cicontinue', None)
         self.payload['prop'] = None
         return ci_list
 
-    def get_duplicatefiles(self, dflimit="max", dfdir="ascending", dflocalonly=None):
+    def get_duplicatefiles(self, dflimit="max", dfdir="ascending", dflocalonly=False):
         prop = 'duplicatefiles'
         self.payload['prop'] = prop
         self.payload['dflimit'] = dflimit
         self.payload['dfdir'] = dfdir
-        if dflocalonly is not None:
+        if dflocalonly==True:
```

Comment:
@ayuhsya The anomalies I noticed were

- for `get_categoryinfo` all the fields are `0`. IMHO it doesn't say anything if the result was the default values if something wrong is queried or they are indeed the correct expected values. Non zero values would be better suited for a test.
- for `get_duplicatefiles` the pageid is `-1`, which is an invalid pageid. If there isn't any pageid for the file. It shouldn't be instantiated as a `WikiPage`.